testing/wireguard-tools/APKBUILD

```diff
@@ -45,7 +43,7 @@ bashcomp() {
 }
 
 tests() {
```

Comment:
This "tests package" simply _will not work_ out of context like this. It does not belong in a distribution package, no matter what it's called. If misunderstandings like this become more widespread than just you, then I wouldn't hesitate to just remove the code from the repository. It really doesn't belong.